_posts/2022-02-17-quarkus-superheroes-to-the-rescue.adoc

```diff
@@ -0,0 +1,239 @@
+
+The Fight service makes outgoing REST calls to both the Hero and Villain services. Calls to the Hero service use the https://quarkus.io/guides/rest-client-reactive[reactive rest client^] while calls to the Villain service use the https://quarkus.io/guides/resteasy-reactive#resteasy-reactive-client[RESTEasy Reactive client^], a https://download.eclipse.org/microprofile/microprofile-rest-client-3.0/microprofile-rest-client-spec-3.0.html[MicroProfile Rest Client^] implementation of the https://docs.oracle.com/javaee/7/tutorial/jaxrs-client001.htm[JAX-RS client API^] that is non-blocking at its core. All outgoing calls use https://quarkus.io/guides/smallrye-fault-tolerance[resilience patterns^] such as https://quarkus.io/guides/smallrye-fault-tolerance#adding-resiliency-retries[retry^], https://quarkus.io/guides/smallrye-fault-tolerance#adding-resiliency-timeouts[timeout^], https://quarkus.io/guides/smallrye-fault-tolerance#adding-resiliency-fallbacks[fallback^], and https://quarkus.io/guides/smallrye-fault-tolerance#adding-resiliency-circuit-breaker[circuit breaking^].
+
+Additionally, the Fight service https://quarkus.io/guides/kafka#integrating-with-kafka-common-patterns[emits^] fight events in https://avro.apache.org/docs/current[Apache Avro^] format to Kafka. The fight schema is automatically registered in an https://www.apicur.io/registry[Apicurio Schema Registry^] by the https://quarkus.io/guides/kafka-schema-registry-avro[Quarkus Apicurio Avro extension^].
+
+The Fight service contains a full suite of unit and integration tests, using https://quarkus.io/guides/getting-started-testing#further-simplification-with-injectmock[mocking^], https://quarkus.io/guides/getting-started-testing#using-spies-instead-of-mocks-with-injectspy[spying^], and testing within https://quarkus.io/guides/hibernate-reactive-panache#transactions[isolated transactions^]. It also makes heavy use of https://wiremock.org/[Wiremock^] for mocking downstream REST endpoints, an https://quarkus.io/guides/kafka#testing-without-a-broker[in-memory Kafka connector^] for performing message verification within unit tests. It also shows how to https://quarkus.io/guides/getting-started-testing#testing-dev-services[inject Dev Services information^] to configure test resources, such as a `+KafkaConsumer+`.
+
+There are four https://quay.io/repository/quarkus-super-heroes/rest-fights?tab=tags[container images available for the Fight service^]: `+java11-latest+`, `+java17-latest+`, `+native-java11-latest+`, and `+native-java17-latest+`.
```

Comment:
does the exact "version" names used in images matter here in the blog introducing them? 

Is it not sufficient to say there are builds available for both Java 11 and 17 and their native variants ?

Reply:
Subsequent PR fixes this